src/ggml.c

```diff
@@ -320,6 +320,16 @@ static ggml_fp16_t ggml_table_exp_f16[1 << 16];
 // precomputed f32 table for f16 (256 KB) (ggml-impl.h)
 float ggml_table_f32_f16[1 << 16];
 
+GGML_CALL const char * ggml_status_to_string(enum ggml_status status) {
```

Comment:
I have a feeling that these `GGML_CALL` are creeping everywhere because nobody knows exactly where they are needed or not.
```suggestion
const char * ggml_status_to_string(enum ggml_status status) {
```